test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
```

Comment:
> The reason there is debug_inspector's RPM package in Fedora (Linux distribution) Project.
I am running this test files without Bundler.

I don't think this is necessary. 
What if bundler is added as a development dependency to the gemspec?